shared/constants/wallets.js

```diff
@@ -532,6 +532,9 @@ export const getAccounts = (state: TypedState) => state.wallets.accountMap.value
 
 export const getSelectedAccount = (state: TypedState) => state.wallets.selectedAccount
 
+export const getSelectedAccountData = (state: TypedState) =>
+  state.wallets.accountMap.get(getSelectedAccount(state), makeAccount())
```

Comment:
should use unknownAccount so its not a new object every time

Reply:
ah forgot about that, I checked for `emptyAccount`